docs/check_orphans.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+
+# This script alerts of orphan files in the modules/ROOT/pages directory
+# that are not referenced in either ./k8up.adoc or modules/ROOT/nav.adoc.
+
+from os import listdir
+from os.path import isfile, join
+import re
+
```

Comment:
I would prefer if this file was written in Go (just like generate.go) and not introduce another programming language to the dev stack

Reply:
This is the same file used in other Antora docs at VSHN. As it's part of the documentation stack, I don't think that this is an issue.